spinnman/transceiver.py

```diff
@@ -918,9 +976,37 @@ def get_cpu_information_from_core(self, x, y, p):
         """
         core_subsets = CoreSubsets()
         core_subsets.add_processor(x, y, p)
-        cpu_infos = self.get_cpu_infos(core_subsets)
+        cpu_infos = self._get_cpu_infos(core_subsets)
         return cpu_infos.get_cpu_info(x, y, p)
 
+    def get_region_base_address(self, x, y, p):
+        """
+        Gets the base address of the Region Table
+
+        :param int x: The x-coordinate of the chip containing the processor
+        :param int y: The y-coordinate of the chip containing the processor
+        :param int p: The ID of the processor to get the address
+        :return: The adddress of the Region table for the selected core
+        :rtype: int
+        :raise SpinnmanIOException:
+            If there is an error communicating with the board
+        :raise SpinnmanInvalidPacketException:
+            If a packet is received that is not in the valid format
+        :raise SpinnmanInvalidParameterException:
+            * If x, y, p is not a valid processor
+            * If a packet is received that has invalid parameters
+        :raise SpinnmanUnexpectedResponseCodeException:
+            If a response indicates an error during the exchange
+        """
+        a = self.read_user(x, y, p, 0)
+        core_subsets = CoreSubsets()
+        core_subsets.add_processor(x, y, p)
+        process = GetCPUInfoProcess(self._scamp_connection_selector)
+        cpu_info = process.get_cpu_info(core_subsets)
+        b = cpu_info.get_cpu_info(x, y, p).user[0]
+        assert a == b
+        return a
```

Comment:
Seems odd to go to the trouble of doing this twice!

Reply:
agree debug do it both ways. Removed